src/workingtitle-vcockpits-instruments-navsystems-g1000/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/AS1000/Shared/BaseAS1000.js

```diff
@@ -786,7 +786,7 @@ class AS1000_PFD_AirportInfos extends NavSystemElement {
         this.facilityName = this.gps.getChildById("facilityName");
         this.type = this.gps.getChildById("type");
         this.timeZone = this.gps.getChildById("timeZone");
-        this.runwwayLength = this.gps.getChildById("runwayLength");
+        this.runwayLength = this.gps.getChildById("runwayLength");
```

Comment:
This was an intentional change?

Reply:
It was intentional. The references in lines 838 and 850 were updated accordingly and nothing from the outside accesses it. But I'm fine with keeping the original name, just wanted to get rid of the typo.

Reply:
Oh man, I'm sorry, I misread the diff on this one...   I thought it was introducing a typo.   I must need more caffeine!  :D

This looks great to me, then.  Thanks for the contribution!